Dev TLS Cert and port mapping changes.

DreamWver opened the pull request:
Created new images for proxy and nicsweb with embedded tls certs.

Commit:
Referencing new pre-configured nics/nicsweb:dev-configured image

Commit:
Reverted changes to the nicsweb dev containter image now that we have a real TLS Cert.
Changed front-end proxy port to use the same standard https port as the internal one. Need this due to
shared client and server configuration referencing the url: endpoint.rest=https://www.nicsdev.tabordasolutions.net/em-api/v1

comeara reviewed (changes requested):
It looks like Apache is still trying to redirect to port 3443. I think the port change requires a corresponding change to Apache.

DreamWver commented:
Good Catch. I'll make the change to the nics.conf.sample over on nics-www.

comeara commented on docker-compose.yml:
```diff
@@ -48,7 +48,7 @@ services:
     image: 618209257270.dkr.ecr.us-west-2.amazonaws.com/nics/proxy:latest
     ports:
       - "3000:80"
```
I suggest you also move http to 80 rather than 3000 if you're going to use 443 for https.

comeara commented:
I retested this today and it is still broken due to the fact that the proxy:dev-configured and web:latest images have not been updated.

Commit:
Switched frontend proxy port mapping from 3000:80 to 80:80 to align with the https port change.
| => Remapped the host port for apache (web) to 8085 since it was conflicting with unchangeable services on the devs machines

DreamWver commented:
I updated the proxy:dev-configured and web:latest images. And changed the port mapping on the proxy as you requested. Also made one more port mapping change that was plaguing us due to a service conflict.
